src/main/java/edu/kit/hci/soli/controller/LoginControllerAdvice.java

```diff
@@ -50,14 +51,7 @@ public LoginStateModel getLoginStateModel(Principal principal, @AuthenticationPr
 
         String id = "kit/" + principal.getName();
         User user = userService.findByUserId(id);
```

Comment:
We should be able to have the `User` injected into this method, meaning this would be unnecessary.

Reply:
I will remove the getLoginStateModel method entirely, since we will not need it anymore